backend/app/testbot/llm.py

```
@@ -2,69 +2,428 @@
-                "maxOutputTokens": 1024,
-                "topP": 0.95,
-                "topK": 40
+            "inputs": prompt,
+            "parameters": {
+                "max_length": 100,
+                "temperature": 0.95,
+                "top_p": 0.9
             }
         }
         
+        response = requests.post(
```

Ruff (0.8.2)

Probable use of `requests` call without timeout

(S113)